<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="WS-195" failures="1" tests="2" name="Default test(failed)" time="0.05" errors="0" timestamp="2019-12-03T18:27:54 BDT">
  <testcase classname="TestNG_ReRunTests.TestNG_ReRunTestsDemo" name="test3" time="0.009">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at TestNG_ReRunTests.TestNG_ReRunTestsDemo.test3(TestNG_ReRunTestsDemo.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="TestNG_ReRunTests.TestNG_ReRunTestsDemo" name="test2" time="0.001"/>
  <testcase classname="TestNG_ReRunTests.TestNG_ReRunTestsDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
